css/main.css

```
@@ -8,3 +8,47 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+
+button {
+  width: 100px;
+  height: 15px;
+  font-family: Poppins;
+  font-size: 10px;
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.6;
+  letter-spacing: normal;
+  text-align: left;
+  color: #4e4e4e;
+  display: flex;
+  
+  
+}
+
+.estilo, 
+.estilo-1{
+  width: 120px;
+  height: 24px;
+  border-radius: 4px;
+  border: solid 1px #e1e1e1;
+  background-color: #ffffff;
+  justify-content: center;
+  
+}
+button i {
+margin-right: 10px;
+}
+.pai {
```

Essa nome não é bom. 
Precisa ser algo mais claro, que tenha o nome daquilo que você quer passar,
```suggestion
.grupo-botoes 
```